lib/license_finder/package_managers/nuget.rb

```diff
@@ -89,6 +89,23 @@ def package_management_command
       "mono #{nuget_binary}"
     end
 
+    def prepare
+      _, stderr, status = Dir.chdir(project_path) { Cmd.run(prepare_command) }
+      return if status.success?
+      log_errors stderr
+      logger.info prepare_command, 'trying fallback prepare command', color: :magenta
+
+      cmd = "#{prepare_command} -PackagesDirectory ."
```

Comment:
It would be nice if you could check the error message for the exact thing you're trying to fix:
`Please specify either -PackagesDirectory or -SolutionDirectory.` before executing this fallback. Unless you think this fallback is a useful general solution.

Reply:
Yes, we could. My concern is that if they change that string, we may not notice and we will still use the fallback.  But maybe we could look for keywords, such as `-PackagesDirectory` or something maybe?

Reply:
Yeah that sounds good